matrix_sdk_base/src/client.rs

```diff
@@ -241,6 +241,19 @@ impl Client {
         Ok(!self.needs_state_store_sync.load(Ordering::Relaxed))
     }
 
+    /// When a client is provided the state store will load state from the `StateStore`.
+    ///
+    /// Returns `true` when a state store sync has successfully completed.
+    pub async fn store_room_state(&self, room_id: &RoomId) -> Result<()> {
+        if let Some(store) = self.state_store.read().await.as_ref() {
+            if let Some(room) = self.get_joined_room(room_id).await {
+                let room = room.read().await;
+                store.store_room_state(room.deref()).await?;
+            }
+        }
+        Ok(())
+    }
```

Comment:
Should we store invited and left rooms also?

Reply:
Yes we should.